src/sage/combinat/cluster_algebra_quiver/cluster_seed.py

```diff
@@ -3326,10 +3331,10 @@ def mutation_class_iter(self, depth=infinity, show_depth=False,
         INPUT:
 
         - ``depth`` -- (default: infinity) integer or infinity, only seeds with distance at most ``depth`` from ``self`` are returned.
-        - ``show_depth`` -- (default: False) if True, the current depth of the mutation is shown while computing.
-        - ``return_paths`` -- (default: False) if True, a shortest path of mutations from ``self`` to the given quiver is returned as well.
-        - ``up_to_equivalence`` -- (default: True) if True, only one seed up to simultaneous permutation of rows and columns of the exchange matrix is recorded.
-        - ``sink_source`` -- (default: False) if True, only mutations at sinks and sources are applied.
+        - ``show_depth`` -- (default: ``False``) if ``True``, the current depth of the mutation is shown while computing.
```

Comment:
could be formatted in 80 columns mode

Reply:
I have made the suggested changes (and more) in #35742 (65e2568628)